drainer/config.go

```diff
@@ -349,3 +347,22 @@ func (cfg *Config) adjustConfig() error {
 
 	return nil
 }
+
+func validateAddr(addr string, strict bool) error {
+	urllis, err := url.Parse(addr)
+	if err != nil {
+		return errors.Annotate(err, fmt.Sprintf("failed to parse addr %v", addr))
```

Comment:
`Annotatef`, not `Annotate` with `Sprintf`...

Reply:
Fixed.